<launch>

  <node pkg="albatros_motor_board" type="motor_board_node" name="motor_board_node" output="screen">

    <param name="serial_port" value="/dev/ttyS0" type="string" />

    <param name="rate_speeds" value="10.0" type="double" />
    <param name="rate_status" value="1.0" type="double" />
    <param name="rate_pressure" value="10.0" type="double" />

    <param name="pressure_offset" value="0" type="int" />
    <param name="waterin_offset" value="0" type="int" />

    <param name="forward_left_PID_Kd" value="0.0" type="double" />
    <param name="forward_left_PID_Ki" value="0.1" type="double" />
    <param name="forward_left_PID_Kp" value="0.5" type="double" />
    <param name="forward_left_PID_on" value="true" type="bool" />
    <param name="forward_left_accel" value="5" type="int" />
    <param name="forward_left_invert" value="false" type="bool" />

    <param name="forward_right_PID_Kd" value="0.0" type="double" />
    <param name="forward_right_PID_Ki" value="0.1" type="double" />
    <param name="forward_right_PID_Kp" value="0.5" type="double" />
    <param name="forward_right_PID_on" value="true" type="bool" />
    <param name="forward_right_accel" value="5" type="int" />
    <param name="forward_right_invert" value="true" type="bool" />

    <param name="downward_left_PID_Kd" value="0.0" type="double" />
    <param name="downward_left_PID_Ki" value="0.1" type="double" />
    <param name="downward_left_PID_Kp" value="0.5" type="double" />
    <param name="downward_left_PID_on" value="true" type="bool" />
    <param name="downward_left_accel" value="5" type="int" />
    <param name="downward_left_invert" value="false" type="bool" />

    <param name="downward_right_PID_Kd" value="0.0" type="double" />
    <param name="downward_right_PID_Ki" value="0.1" type="double" />
    <param name="downward_right_PID_Kp" value="0.5" type="double" />
    <param name="downward_right_PID_on" value="true" type="bool" />
    <param name="downward_right_accel" value="5" type="int" />
    <param name="downward_right_invert" value="true" type="bool" />

  </node>

  <node pkg="dynamic_reconfigure" type="reconfigure_gui" name="dynamic_reconfigure_node" output="screen" />

</launch>
